src/components/shared/levelSelect.tsx

```diff
@@ -34,6 +39,22 @@ function LevelSelect(props:LevelSelectProps): JSX.Element {
   };
 
   useEffect(() => {
+    const num = Number(localStorage.getItem('highestPage'));
+    setHighestPage(num || -1);
+  }, []);
+
+  useEffect(() => {
+    if (isCompleted) {
+      setHighestPage(pageOptions.indexOf(currentPage) + 1);
```

Comment:
When a user goes back a few pages then moves forward using the next button it will cause the highest page to be the page they just moved to which doesn’t make sense since it would be lower. Checking `currrentPage` for that here should fix the issue.